audit/projects/k8s-conform/buckets/k8s-conform-cri-o/iam.json

```diff
@@ -16,7 +16,8 @@
     },
     {
       "members": [
-        "group:k8s-infra-conform-cri-o@kubernetes.io"
+        "group:k8s-infra-conform-cri-o@kubernetes.io",
+        "serviceAccount:service-cri-o@k8s-conform.iam.gserviceaccount.com"
```

Comment:
Everything else as of this commit reflects a run of `ensure-conformance-storage.sh` that did _not_ have the env var set to disable extraneous services, and looks expected to me